Makefile

```diff
@@ -43,7 +43,9 @@ macpoetry-install:  # Same as 'poetry install' except that on OSX Catalina, an e
 
 configure:  # does any pre-requisite installs
 	pip install --upgrade pip
-	pip install poetry==1.1.4  # this version is known to work, so we'll use it for now. -kmp 3-Mar-2021
+	pip install poetry==1.1.9  # this version is known to work, so we'll use it for now. -kmp 3-Mar-2021
```

Comment:
I would have just left it open, since 1.1.9 is latest and I'd _like_ to believe that it's now going to be maintained going forward, but I can live with this for now.